.github/workflows/tests-ci.yml

```diff
@@ -106,12 +106,6 @@ jobs:
         with:
           cache: "true"
 
-      - name: Print Node.js, npm, & pnpm versions for debugging if needed
-        run: |
-          node -v
-          npm -v
-          pnpm -v
-
```

Comment:
For my learning: this was added recently to debug something (I don't recall what), judging from the removal, you probably concluded this is unnecessary, can you elaborate a bit?

Reply:
this particular ci workflow is time sensitive. i tuned it to be as fast as possible in the past. so adding a bunch of random stuff that's on net not necessary hinders keeping it fast. plus worth pointing out all of these versions are well known throughout multiple places in the repo (like for example the hermit folder)

Reply:
Fair point about hermit folder. Me dumb.